package.json

```diff
@@ -34,12 +34,13 @@
     "react": "16.9.0",
     "react-native": "https://github.com/expo/react-native/archive/sdk-36.0.1.tar.gz",
     "react-native-dropdownalert": "^4.1.0",
-    "react-native-gesture-handler": "~1.5.0",
+    "react-native-gesture-handler": "^1.5.6",
     "react-native-hyperlink": "0.0.19",
     "react-native-modal": "^11.1.0",
-    "react-native-reanimated": "~1.4.0",
+    "react-native-reanimated": "^1.4.0",
     "react-native-safe-area-context": "^0.7.3",
     "react-native-safe-area-view": "^1.0.0",
+    "react-native-screens": "^2.2.0",
```

Comment:
what's this?? 🤔

Reply:
@dereckquock it fixed an issue compiling for some reason. Let me try to remove it and see if it still works.

Reply:
Unable to resolve module `react-native-screens` from `node_modules/react-navigation-stack/lib/module/views/StackView/StackViewCard.js`: react-native-screens could not be found within the project.

If you are sure the module exists, try these steps:
 1. Clear watchman watches: watchman watch-del-all
 2. Delete node_modules: rm -rf node_modules and run yarn install
 3. Reset Metro's cache: yarn start --reset-cache
 4. Remove the cache: rm -rf /tmp/metro-*

RCTFatal
__28-[RCTCxxBridge handleError:]_block_invoke
_dispatch_call_block_and_release
_dispatch_client_callout
_dispatch_main_queue_callback_4CF
__CFRUNLOOP_IS_SERVICING_THE_MAIN_DISPATCH_QUEUE__
__CFRunLoopRun
CFRunLoopRunSpecific
GSEventRunModal
UIApplicationMain
main
start

Reply:
🤷‍♂ ok, just keep it then!